lib/utils.ts

```diff
@@ -452,8 +458,8 @@ async function requestAsync(
 	let timerId: ReturnType<typeof setTimeout> | undefined;
 	try {
 		const requestTime = Date.now();
-		let p = fetch(url, opts);
-		if (opts.timeout && IS_BROWSER) {
+		let p = $fetch(url, opts);
+		if (opts.timeout) {
```

Comment:
This adds a new functionality which is timeouts on non-browser requests (which wouldn't be respected previously)

Reply:
Afair this was working already before, and I had to add this to keep the tests happy once I moved to fetch.
So I guess it was working b/c of node-fetch supporting it while the fetch ponyfill wasn't.